### ISSUE TITLE
Feature: Read passwords from file (docker secrets)

### ISSUE THREAD

**Issue by jalr** [mongo-express/mongo-express#363]
As I don't like passing secrets via environment variables and docker-compose now supports secrets which mount files for passing passwords securely I would suggest to use this approach for 
```ME_CONFIG_BASICAUTH_PASSWORD```, ```ME_CONFIG_MONGODB_AUTH_PASSWORD``` and ```ME_CONFIG_MONGODB_ADMINPASSWORD```

On the docker hub I saw a lot of images using the __FILE_ suffix for providing the path to the mounted secret. This is my current implementation:

```
var fs = require('fs');
```
```
adminPassword: fs.readFileSync(process.env.ME_CONFIG_MONGODB_ADMINPASSWORD_FILE).toString().split(/\r?\n/)[0].trim(),
```

To support both methods we could check if the __PASSWORD_FILE_ variable is set and if not, fallback to the __PASSWORD_

**Comment by positron96** [comment]
What's the status of this proposal? Would be very useful to have it.

**Comment by dozoisch** [comment]
@positron96 you're welcome to make a PR. I don't think anyone is currently working on this.

**Comment by positron96** [comment]
I'd like to, but me no JS developer.

**Comment by dozoisch** [comment]
@positron96 unfortunately, I don't have the bandwidth to take care of this.

**Comment by jeremydouglass** [comment]
Can this be closed? It looks like it was merged in #460, and then deployed to docker in https://github.com/mongo-express/mongo-express-docker/issues/29

**Comment by knickers** [comment]
Yeah, it looks like everything's done. I'm good with closing this :+1:
